crates/nu-cli/src/commands/autoview.rs

```diff
@@ -179,7 +179,13 @@ pub fn autoview(context: RunnableContext) -> Result<OutputStream, ShellError> {
                                 value: UntaggedValue::Primitive(Primitive::Decimal(n)),
                                 ..
                             } => {
-                                out!("{}", n);
+                                // TODO: normalize decimal to remove trailing zeros.
+                                // normalization will be available in next release of bigdecimal crate
+                                let mut output = n.to_string().trim_end_matches('0').to_owned();
```

Comment:
You'll probably want to make sure you do this only if there's a `.` in there, otherwise `10000` → `1` 🙂

Reply:
I think checking if there is `.` in there is not needed because when we pass any digit(of course I'm assuming that we are using `str --to-float`), it is parsed as decimal and that dot is already there in string representation. So `echo "10000" | str --to-float` replies with 10000.0 which is correct

Reply:
It might be good idea to add test like that, so when we change it to proper normalization when there will be no surprise

Reply:
I think you'll still want such a check here though, because decimal values could be formed from commands other than `str --to-float`. If you have the following:

```rust
    use std::str::FromStr;
    println!("{}", bigdecimal::BigDecimal::from_str("1000").unwrap());
```

You'll see the output `1000`. I don't know what other commands do that right now, but I'd rather be safe than sorry 🙂